.github/workflows/publish.yml

```diff
@@ -0,0 +1,21 @@
+name: Publish
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch: {}
+
+jobs:
+  publish:
+    uses: wetransform/gha-workflows/.github/workflows/sbt-library-check.yml@master
+    with:
+      java-version: 17
+      expect-tests: true
+    secrets:
+      WETF_ARTIFACTORY_USER:
```

Comment:
Example on how secrets need to be passed:
https://github.com/wetransform-os/topojson-j/blob/864f8617245363957a5caa42ea222eb4e6bbc600/.github/workflows/publish.yml#L15